steps/close-jira-tasks/1.0.0/step.yml

```diff
@@ -0,0 +1,100 @@
+title: close-jira-tasks
+summary: |
+  Step moves Jira tasks from given initial state to target state if their ids are in git log
+description: |
+  Step searches Jira project for tasks that are in `from_status` (e.g. `Deploying`) state
+  and after successful build moves them to `to_status` (e.g. `Stabilization on QA`) state.
+
+  This is useful if you use smart commits in Jira, e.g. if your commit message
+  includes `TEST-123 #Do`.
+  This would cause the issue TEST-123 to transition according to `Do` keyword.
+
+  The step works for multiple issues mentioned in last 100 commits, so if
+  you finished multiple tasks it should move all of them to the `to_status` state
+  provided they are in `from_status` at the moment of execution.
+
+  The step can also add custom value to the custom field in the Jira issue e.g. version of the app.
+website: https://github.com/orestesgaolin/bitrise-step-close-jira-tasks
```

Comment:
This URL looks outdated. Shouldn't it be https://github.com/orestesgaolin/bitrise-close-jira-tasks ?